packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -202,7 +202,11 @@ const MeetingUtil = {
         meeting.reconnectionManager.cleanUp();
       })
       .then(() => meeting.stopKeepAlive())
-      .then(() => meeting.updateLLMConnection());
+      .then(() => {
+        if (meeting.config.enableAutomaticLLM) {
```

Comment:
```suggestion
        if (meeting?.config?.enableAutomaticLLM) {
```

Reply:
Can meeting and meeting.config be undefined here?

Reply:
added for the config as for meeting it wasn;t required

Reply:
meeting is not undefined but config can be